src/main/java/br/unb/cic/analysis/df/OverridingAssignmentAnalysis.java

```diff
@@ -46,13 +46,10 @@ public OverridingAssignmentAnalysis(Body methodBody, AbstractMergeConflictDefini
 
                 for (DataFlowAbstraction defsIn: in){
                     String defInName = getLocalName(defsIn.getLocal());
-
-                    for (Local useBox: getUseVariables(u)){
-                        String useBoxName = getLocalName(useBox);
-                        if (!defInName.equals(useBoxName) && defInName.equals(localName)){
```

Comment:
@barbosamaatheus There is no need to analyze the UseBox of a definition (DefBox). Example: x = ...; no matter what comes on the right of the statement.